manifests/globals.pp

```diff
@@ -2,10 +2,26 @@
 #
 # @param scl
 #   Use a specific Software CoLlection on Red Hat based systems
+#
+# @param version
+#   The version of Redis to install and manage.
+#
+# @param manage_dnf_module
+#   Manage the DNF module. This only makes sense on distributions that use DNF
+#   package manager, such as EL8 or Fedora. It also requires Puppet 5.5.20+ or
+#   Puppet 6.15.0+ since they ship the dnfmodule provider.
```

Comment:
I'd be happy with changing `metadata.json` to 6.15.0+:
https://github.com/voxpupuli/puppet-redis/blob/38b0d2d4bb2abc0806e9f148ce385ae9a28deb38/metadata.json#L28-L33